vita3k/interface.cpp

```diff
@@ -203,7 +203,7 @@ bool install_archive_content(EmuEnvState &emuenv, GuiState *gui, const ZipPtr &z
         }
     }
 
-    if (fs::exists(output_path / "sce_sys/package/")) {
+    if (fs::exists(output_path / "sce_sys/package/") && emuenv.app_info.app_title_id.starts_with("PCS")) {
```

Comment:
```suggestion
    if ((fs::exists(output_path / "sce_sys/package/") && emuenv.app_info.app_title_id.starts_with("PCS")) {
```

you  little forget one things haha
and change commit name, said "add check if is commercial app of decrypt nopdrm" exemple